DOCS-3470 added instructions for approving and deleting access (#304)
mulesoft/docs-api-manager: modules/ROOT/pages/access-client-app-id-task.adoc

include::_attributes.adoc[]
endif::[]

> As an owner of a xref:exchange::to-request-access.adoc[registered client application], you are occasionally required to provide the client ID and the client secret of the application when accessing an API. 

Reviewer: ```suggestion
As the owner of a xref:exchange::to-request-access.adoc[registered client application], you are occasionally required to provide your application's client ID and client secret when accessing an API. 
```
"an" owner or "the" owner of the application? Is there more than one owner?

Author: Done! Thanks!

> As the owner of a xref:exchange::to-request-access.adoc[registered client application], you are occasionally required to provide your client ID and client secret (optional) when accessing an API, after a contract is created between your application and an API. 

Reviewer 2: This paragraph is a bit confusing. If you are creating a Client Application, is probably because the contract is required to consume the API. It is not an optional parameter.

Author: Done! Thanks!

Author: Done! Thanks!

> After a contract is created between the client application and the API, you can access the API using these credentials based on the policies that are applied. For example, if the API owner has applied the Client ID Enforcement policy on the API, you must provide the client ID, and optionally, the client secret information for the client application. 

Reviewer: ```suggestion
After a contract is created between your application and an API, you access that API using these credentials, depending on the policies applied to that API. For example, if the API owner has applied the Client ID Enforcement policy, you must provide your application's client ID and, optionally, its client secret to access the API. 
```
Even edited, this information is repetitive of the previous paragraph and actually backtracks from "a registered client app" to the contract that makes it so. The two paras should be in chrono order.

Reviewer: I am confused about this topic. 
The text makes clear that the user task is to **access the API by using the credentials**, **not to access the credentials themselves.** Please clarify.

Author: Done! Thanks!

> You must have the Organization Administrator privileges of the master organization, or must be the owner of the client application to access the credentials.  

Reviewer: Doesn't the owner already **have** the credentials; didn't the owner **create** the application credentials?

Author: Nope, the credentials are created by AP when a contract is created between the app and the API.

. From the details pane, click the application for which you want to view the client credentials.
+
The client ID and client secret credentials for the selected application appear on the right of the window. 

> . Alternatively, if you need to obtain only the client ID of the registered client application, select the correct version of the API and click *Contracts* from the left navigation. 

Reviewer 2: It may be useful to add as related this Doc
https://docs.mulesoft.com/exchange/about-my-applications

If the user does not have access to API Manager, but he is the Application Owner, he can see the Application Credentials in Exchange.

Author: Done! Thanks!
